doc/cordova.txt

```diff
@@ -23,6 +23,12 @@ Project Commands
     serve .............................. Run project with a local webserver
                                             (including prepare)
 
+Telemetry Commands
+    telemetry on ....................... Turn telemetry collection on
```

Comment:
Sounds like telemetry is a "global command" and should go along with create & help. All details about telemetry command e.g. privacy notice, on/off should not be specified here. The goal of this top level help is to be succinct without going into details.